### PR TITLE
Add -c flag

### PR DESCRIPTION
**Pull request by geek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ This allows you to issue the command `npm run lint`, which will run JavaScript s
 
 `belly-button` accepts the following command line options:
 
+  - `-c` (alias `--config`) - Path to `.eslintrc.js` file to load configuration from.
```
Comment (cjihrig) [inline README.md]: Can we leave this undocumented :-)

File: test/fixtures/config/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
+'use strict';
```
Comment (cjihrig) [inline test/fixtures/config/.eslintrc.js]: Can you extend the base config here and only include the changes.

File: test/cli.js [hunk 3]
```diff
@@ -188,5 +188,36 @@ describe('Belly Button CLI', function () {
         done();
       });
     });
+
+    it('defaults to belly-button style', function (done) {
+      var lintFile = Path.join(__dirname, 'fixtures', 'config', 'yoda.js');
```
Comment (cjihrig) [inline test/cli.js]: Can you use `fixturesDirectory` (defined at the top of the file) instead of `__dirname` and `'fixtures'`.

File: test/cli.js [hunk 4]
```diff
@@ -188,5 +188,36 @@ describe('Belly Button CLI', function () {
...
+
+      child.once('close', function (code, signal) {
+        expect(code).to.equal(1);
+        expect(signal).to.equal(null);
+        done();
+      });
+    });
+
+    it('can override config file', function (done) {
+      var configFile = Path.join(__dirname, 'fixtures', 'config', '.eslintrc.js');
```
Comment (cjihrig) [inline test/cli.js]: Same comment on this line, and the next one.

File: test/fixtures/config/yoda.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+var i = 0;
```
Comment (cjihrig) [inline test/fixtures/config/yoda.js]: Blank line after this line please.

Comment (geek) [inline test/fixtures/config/yoda.js]: done

**Comment by cjihrig** [comment]
Thanks, landed in 72ffb20df11886e3712580dea742ae6df7330338.

**Pull request closed without merging**